generators/service/index.js

```diff
@@ -1,5 +1,5 @@
 /*
- * © Copyright IBM Corp. 2017, 2018
```

Comment:
I think this is supposed to be 2017, 2020

Reply:
This looks outdated, not sure why. I changed it to just 2019 because it's technically a new repo so this file is new and it was made last year.